baselines/teams.md

```diff
@@ -295,40 +320,61 @@ Docs](https://docs.microsoft.com/en-us/microsoftteams/manage-external-access#man
 
 Steps are outlined in [Manage contact with external Teams users not
 managed by an
-organization](https://docs.microsoft.com/en-us/microsoftteams/manage-external-access#manage-contact-with-external-teams-users-not-managed-by-an-organization).
+organization](https://docs.microsoft.com/en-us/microsoftteams/manage-external-access#manage-contact-with-external-teams-users-not-managed-by-an-organization). All the settings in this section are configured in the [**Microsoft Teams admin center**](https://admin.teams.microsoft.com).
 
-1.  Sign in to the **[Microsoft Teams admin
-    center](https://admin.teams.microsoft.com).**
-
-2.  Select **Users -\> External access.**
+#### MS.TEAMS.5.1v1 instructions:
+    
+1.  Sign in to the [**Microsoft Teams admin
+    center**](https://admin.teams.microsoft.com).
+2.  Select **Users > External access.**
 
 3.  To completely block contact with unmanaged users, under **Teams
     accounts not managed by an organization**, set **People in my
     organization can communicate with Teams users whose accounts aren't
     managed by an organization** to **Off**.
-
+    
 4.  To allow contact with unmanaged users only if the internal user
     initiates the contact:
 
     1.  Under **Teams accounts not managed by an organization**, set **People in my organization can communicate with Teams users whose accounts aren't managed by an organization** to **On**.
 
     2.  Clear the check next to **External users with Teams accounts not managed by an organization can contact users in my organization**.
 
+
+#### MS.TEAMS.5.2v1 instructions:
```

Comment:
The policies in Section 5 are confusing and appear to conflict with each other. Let's discuss these verbally. Policy 5.1 seems to have two different compliant configuration states (as noted by instructions steps 3 and 4 which cannot be configured at the same time since they represent two distinct states. Policy 5.2 has a single compliant configuration state (noted by instruction step 3) which is identical to policy 5.1 step 3 and therefore seems duplicative.

Overall, I cannot think of a good business case where you would want a federal agency user to communicate with an unmanaged user (i.e. does not belong to an organization on the approved domain list from policy MS.TEAMS.4.1v1). We can discuss so I can hear your feedback from the pilots and ideas.  

I believe there should only be a single compliant policy / configuration: Completely block contact with unmanaged users: under **Teams accounts not managed by an organization**, set **People in my organization can communicate with Teams users whose accounts aren't managed by an organization** to **Off**.